README.md

```diff
@@ -122,14 +124,22 @@ To install `Ubuntu Make`, run the following command:
 
 	   
 ## 2. Installing the resource aggregator for ODIM and GRF plugin
-This section provides a step-by-step procedure for deploying the resource aggregator for ODIM (odimra) and GRF plugin.
+This section provides a step-by-step procedure for deploying ODIMRA and GRF plugin.
 
   
   **NOTE:**
   - All configuration parameters are set to default values in the configuration files for odimra and GRF plugin. 
   - The following ports are used for deploying odimra and GRF plugin:
     45000, 45001, 45101-45110, 9092, 9082, 6380, 6379, 8500, 8300, 8302, 8301, 8600
     Ensure that the above ports are not in use.
+  - The following users are created and added to group Ids automatically during deployment.
+  
+    |User Id| Group Id|
+	-----|---------|
+	|`odimra`|1234 |
+	|`plugin`|1235 |
+	
+    These users are created on both the VM and the containers for the resource aggregator and the plugin. Ensure not to create them again.
```

Comment:
User Id "odimra" & Group Id "1234" will be created on the VM and in the odimra container.
User Id "plugin" & Group Id "1235" will be created on the VM and in the grfplugin container.
Ensure  these User Id & Group Id are not present on the VM before deploying ODIMRA.